### PR TITLE
Add token generator and Sync grant as part of it

### PR DESCRIPTION
**Pull request by mamont**

### CODE REVIEW

File: Twilio/AccessToken.php [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    {
+        $header = array(
+            'cty' => 'twilio-fpa;v=1',
+            'typ' => 'JWT'
+        );
+
+        $now = time();
+
+        $grants = array();
+        if ($this->identity) {
```
Comment (ajtack) [inline Twilio/AccessToken.php]: Note: This condition is incorrect, needs to be `if (!is_null($this->identity))`.

**Comment by ajtack** [comment]
This needs to be fit in with the style of the next-gen library. Cherry-pick c4b4a02c59d9418a34460cf8c5599668501ca771?

**Comment by mamont** [comment]
Cherry-picked, thanks

**Comment by ajtack** [comment]
To be closed -- replaced by #346.

**Pull request closed without merging**